beacon_node/network/src/sync/manager.rs

```diff
@@ -258,10 +260,11 @@ pub fn spawn<T: BeaconChainTypes>(
     network_send: mpsc::UnboundedSender<NetworkMessage<T::EthSpec>>,
     beacon_processor: Arc<NetworkBeaconProcessor<T>>,
     sync_recv: mpsc::UnboundedReceiver<SyncMessage<T::EthSpec>>,
+    fork_context: Arc<ForkContext>,
```

Comment:
nit: we can just pass the `ForkName` which is `Copy` and avoid clonning the `Arc` when passing it

Reply:
`current_fork` can change during runtime so we may use the outdated value if we do that . its just an Arc clone so it's okay imo

Reply:
makes sense, thanks for explaining Pawan!